alpaca/broker/models/trading.py

```diff
@@ -9,4 +9,4 @@ class Order(BaseOrder):
         commission (float): The dollar value commission you want to charge the end user.
     """
 
-    commission: float
+    commission: float | None = None
```

Comment:
Small change but we should use the following syntax for optional parameters

`commission: Optional[float] = None`

Reply:
Changed in the latest commit